pyproject.toml

```diff
@@ -14,10 +28,11 @@ requires-python = ">= 3.8"
 requires = ["hatchling"]
 build-backend = "hatchling.build"
 
-[tool.rye]
+[project.scripts]
+build = "mitol.build_support.cli:cli"
+
+[tool.uv]
 managed = true
-virtual = true
```

Comment:
Is there a reason that you opted to not use the virtual workspace functionality and add the packages as dependencies? The corresponding syntax is `tool.uv.package = false` https://docs.astral.sh/uv/concepts/projects/#configuring-project-packaging

Reply:
I missed it reviewing the docs. FWIW, I tried it and I still had to have the dependencies set up to make the test scaffolding work (though I didn't spend much time on it).